### ISSUE TITLE
New materials panel bug killing GUI

### ISSUE THREAD

**Issue by joelvbernier** [HEXRD/hexrdgui#843]
On my Mac, I am now getting the following when I try to start the GUI:

```
Traceback (most recent call last):
  File "/Users/bernier2/miniconda3/envs/hexrdgui/bin/hexrdgui", line 33, in <module>
    sys.exit(load_entry_point('hexrdgui', 'gui_scripts', 'hexrdgui')())
  File "/Users/bernier2/Documents/GitHub/hexrdgui/hexrd/ui/main.py", line 30, in main
    window = MainWindow()
  File "/Users/bernier2/Documents/GitHub/hexrdgui/hexrd/ui/main_window.py", line 83, in __init__
    self.add_materials_panel()
  File "/Users/bernier2/Documents/GitHub/hexrdgui/hexrd/ui/main_window.py", line 264, in add_materials_panel
    self.materials_panel = MaterialsPanel(self.ui)
  File "/Users/bernier2/Documents/GitHub/hexrdgui/hexrd/ui/materials_panel.py", line 29, in __init__
    self.materials_table = MaterialsTable(m, parent=self.ui)
  File "/Users/bernier2/Documents/GitHub/hexrdgui/hexrd/ui/materials_table.py", line 33, in __init__
    self.update_material_name()
  File "/Users/bernier2/Documents/GitHub/hexrdgui/hexrd/ui/materials_table.py", line 61, in update_material_name
    self.ui.setWindowTitle(f'{self.title_prefix}{self.material.name}')
  File "/Users/bernier2/Documents/GitHub/hexrd/hexrd/material.py", line 781, in _get_name
    return self._name
AttributeError: 'Material' object has no attribute '_name'
```

@psavery could this be related to #834?  High priority bug.

**Comment by joelvbernier** [comment]
Actually, @saransh13, did you get rid of the `_name` attribute in your latest hexrd PR?

**Comment by psavery** [comment]
I think this got fixed by some hexrd PRs, such as hexrd/hexrd#235 and hexrd/hexrd#236
